README.md

```diff
@@ -27,13 +27,14 @@
     /></a>
 </p>
 
-`bon` is a Rust crate for generating compile-time-checked builders for functions and structs.
+
+`bon` is a Rust crate for generating compile-time-checked builders for functions and structs, supporting partial struct building and idiomatic partial application with optional and named parameters for functions and methods.
 
 Visit the [guide for a complete overview of the crate](https://elastio.github.io/bon/guide/overview).
 
 ## Quick examples
 
-`bon` can turn a function with positional parameters into a function with "named" parameters via a builder. It's as easy as placing the `#[builder]` attribute on top of it.
+`bon` turns a function with positional parameters into a function with named parameters just by placing the `#[builder]` attribute on top of it.
```

Comment:
`just by` placing does sound right as if `bon` is the one who palces the `#[builder]` attribute when in fact, it's the developer who does this. Anyway, I'll make my corrections to the readme in #113